debug.log

```diff
@@ -0,0 +1 @@
+[1013/123951.528:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
Cypress debug logs sometimes change, I don't think they are supposed to be committed. Lets remove this from the pull request.

Reply:
Should I add this file to git ignore?